src/server/utilities/liteATIClickTracking/index.test.ts

```diff
@@ -0,0 +1,113 @@
+import trackingScript from '.';
+
+const dispatchClick = (targetElement: HTMLElement) => {
+  document.body.appendChild(targetElement);
+  const event = new MouseEvent('click', {
+    view: window,
+    bubbles: true,
+    cancelable: true,
+  });
+  targetElement.dispatchEvent(event);
+};
+
+describe('Click tracking script', () => {
+  const randomUUIDMock = jest.fn();
+  beforeEach(() => {
+    jest.clearAllMocks();
+    jest.useFakeTimers().setSystemTime(new Date(1731515402000));
+
+    trackingScript();
+
+    document.cookie =
+      'atuserid=; expires=Thu, 01 Jan 1970 00:00:00 UTC; path=/;';
+
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
```

Comment:
This can be replaced with 
```suggestion
    // @ts-expect-error Explain why here
```